src/utilities/layout-engine/vertical-tree.js

```diff
@@ -21,12 +21,24 @@ import SnapToGrid from './snap-to-grid';
 
 class VerticalTree extends SnapToGrid {
   adjustNodes(nodes: INode[], nodesMap?: any): INode[] {
-    const { nodeKey, nodeSize } = this.graphViewProps;
-    const size = (nodeSize || 1) * 1.5;
+    const { nodeKey, nodeSize, nodeHeight, nodeWidth, nodeSpacingMultiplier } = this.graphViewProps;
     const g = new dagre.graphlib.Graph();
     g.setGraph({});
     g.setDefaultEdgeLabel(() => ({}));
 
+    const spacing = nodeSpacingMultiplier || 1.5
```

Comment:
Add semicolon

Reply:
I've created an issue to correct stylistic problems such as this with eslint rules. I'll update the code to automatically test and correct these problems at a later date. For now this code should build correctly so I won't worry too much about it.